### PR TITLE
#368 Clarified the use case for the squbs marshalling library.

### PR DESCRIPTION
**Pull request by akara**
Thanks for your pull request.  Please review the following guidelines.

- [X] Title includes issue id.
- [X] Description of the change added.
- [X] Commits are squashed.
- [ ] Tests added.
- [X] Documentation added/updated.
- [X] Also please review [CONTRIBUTING.md](https://github.com/paypal/squbs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: docs/marshalling.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ Marshalling and unmarshalling is used both on the client and server side. On the
...
 
-squbs provides a Java API for manual marshalling/marshalling as well as adding facilities for working in a Scala/Java cross-language environment. Manual access to marshallers and unmarshallers is useful for stream-based applications where some work may need to be done in a stream stage. It is also useful for testing marshaller configurations to ensure the right format is achieved.
+squbs provides marshallers for cross-language environments. For instance, when working with a mixed object hierarchy having both Scala case classes as well as Java beans. For simple, single-language environments, please use the provided marshallers or other open source marshallers directly.
```
Comment (anilgursel) [inline docs/marshalling.md]: The above links provide details about marshalling/unmarshalling design, which is useful.  However, for the users who would likely to immediately get a json marshalling/unmarshalling example, I believe they remain little complicated.  Accordingly, should we add a link to http://doc.akka.io/docs/akka-http/current/scala/http/common/json-support.html and its Java version?

**Comment by akara** [comment]
I noticed the "##" issue. But that's throughout our documentation. I'm wondering whether this is a temporary bug they have or is it going to be permanent. Let me check.

**Comment by anilgursel** [comment]
I remember seeing an email recently about Github starting to follow a spec for markdown..  Not sure it is related to that at all..

**Comment by anilgursel** [comment]
I was referring to https://github.com/blog/2333-a-formal-spec-for-github-flavored-markdown

**Pull request merged**